pkg/machine/hyperv/stubber.go

```diff
@@ -547,3 +547,16 @@ func createNetworkUnit(netPort uint64) (string, error) {
 	netUnit.Add("Install", "WantedBy", "multi-user.target")
 	return netUnit.ToString()
 }
+
+func (h HyperVStubber) ConnectionInfo(name string) (*define.VMFile, *define.VMFile, error) {
```

Comment:
would it make sense to hand this off of MachineConfig ?

Reply:
Yes, I think that would be better. It would address the code duplication that @vrothberg mentioned above.